src/Model/ManagerInterface.php

```diff
@@ -56,30 +58,30 @@ public function findOneBy(array $criteria, array $orderBy = null);
      *
      * @param mixed $id The identifier
      *
-     * @return object
+     * @return object|T
```

Comment:
```suggestion
     * @return object|T|null
```
Shouldn't be this null as well?

Reply:
You're right. Doctrine could return `null`